io/io/inc/TEmulatedCollectionProxy.h

```diff
@@ -64,12 +64,24 @@ class TEmulatedCollectionProxy : public TGenCollectionProxy  {
    // Virtual in-place constructor
    virtual void* New(void* memory)   const {  return new(memory) Cont_t; }
 
+   // Virtual constructor
+   virtual TClass::ObjectPtr NewObject()   const             {  return new Cont_t;         }
```

Comment:
Here and below, can the `ObjectPtr` construction be made more explicit? E.g.
```suggestion
   virtual TClass::ObjectPtr NewObject()   const             {  return {new Cont_t, nullptr};         }
```

Reply:
sure.